packages/field-base/test/field-mixin.test.js

```diff
@@ -833,6 +833,15 @@ const runTests = (defineHelper, baseMixin) => {
         await nextFrame();
         expect(input.getAttribute('aria-labelledby')).to.be.equal('accessible-name-ref-0');
       });
+      it('should change aria-labellebdy when accessibleNameRef is set before element is attached', async () => {
```

Comment:
Looks like it could be worth a separate test suite.

```js
describe('accessible-name-ref is set before attached to the DOM', () => {
  ...
})
```

Reply:
Done. Also made the same change to another similar test for `accessible-name`

Reply:
One clarification: I meant to move it outside the `accessible-name-ref` test suite so that their beforeEach don't interfere with each other.

Reply:
Or probably it could be simplified like that:

```js
describe('accessible-name-ref is set initially', () => {
  beforeEach(async () => {
     element = fixtureSync(`<${tag} label="Label" accessible-name-ref="..."></${tag}>`);
     await nextRender();
     input = element.querySelector('[slot=input]');
  });

  it('should...', () => {
  
  });
});
```

Reply:
I think it's done now 😅